sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -388,14 +388,15 @@ private[ui] class ExecutionPagedTable(
         +details
       </span> ++
       <div class="stage-details collapsed">
-        <pre>{execution.details}</pre>
+        <pre>{execution.description}<br></br>{execution.details}</pre>
```

Comment:
why is this change?

Reply:
@cloud-fan you can see the attached snapshot of after change in UI. With this change we can see SQL and details in the details box ``+details``

Reply:
https://github.com/apache/spark/pull/23958#issuecomment-471156490

Reply:
does it introduce changes to non-SQL queries? Sorry I'm not familiar with the frontend part.

Reply:
non sql queries will have callsite.short form instead of sql which is one liner usually

Reply:
even without this change, we can still see the full SQL when clicking the `+details`, right? IIUC the full SQL is in `execution.details` not `execution.description`

Reply:
No. You can refer at ``org.apache.spark.sql.execution.ui.SQLAppStatusListener#onExecutionStart`` here you can see description is copied from ``SparkListenerSQLExecutionStart.description`` 

Refer code @ ``org.apache.spark.sql.execution.SQLExecution#withNewExecutionId``
``SparkListenerSQLExecutionStart.description`` maps to ``execution.description`` -- contain full SQL or callsite.shortform
``SparkListenerSQLExecutionStart.details`` maps to ``execution.details `` -- contains callsite.longform

Reply:
How can we know the full SQL if it's truncated in `SQLExecution#withNewExecutionId`?

Reply:
We cannot know. There are two truncations here 
1. Backend - Done @ ``SQLExecution#withNewExecutionId`` this is controlled by config and if exceeds threshold string will be truncated before posting ``SparkListenerSQLExecutionStart``.
2. UI - This gets input via ``SparkListenerSQLExecutionStart`` from status store (truncated or may not be truncated) but this may need additional truncation based on user window/browser size. this is done by css - onload or even window resize will trigger this truncation. This is not configurable. This is adjusted as per window size on runtime by css

Reply:
Refer https://github.com/apache/spark/pull/23958#discussion_r264091072

Reply:
ah now I got it, thanks for the explanation!